tasks/debian.yml

```diff
@@ -1,33 +1,28 @@
 ---
 # tasks file for ansible-role-squid
 
-- name: "Install squid"
-  apt: name=squid state=latest
-
-- name: "Install iproute2"
-  apt: name=iproute2 state=latest
+- name: "Install iproute2 on Debian"
+  apt:
+    name: iproute2
+    state: "{{ squid_package_state }}"
 
 - name: "Set up extra cache dir"
-  file: path={{ squid_cache_dir2["path"] }} state=directory owner=proxy group=proxy mode=0755
+  file:
+    path: "{{ squid_cache_dir2["path"] }}"
```

Comment:
Accordingly to travis this is broken to many """ so that travis does not know what belongs to what. I think travis is correct.

Reply:
Well spotted. I agree. Let's see if this other syntax works.